big_tests/tests/login_SUITE.erl

```diff
@@ -245,6 +242,8 @@ verify_format(GroupName, {_User, Props}) ->
     SPassword = rpc(mim(), ejabberd_auth, get_password, [Username, Server]),
     do_verify_format(GroupName, Password, SPassword).
 
+do_verify_format(login_scram, _Password, #{salt := _S, iteration_count := _IC}) ->
```

Comment:
Shouldn't we expect the sha/sha256 key in this map?

Reply:
Depending on the setup we are to expect one or both of them. The goal of this PR is to allow to easily extend the module by adding further sha. The key could be any combination of allowed sha. The thing that is to be constant is the salt and iteration_count. That is why we do pattern matching on the things that we expect to always be there.

Reply:
ok